server.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+
+const finalhandler = require('finalhandler');
+const bodyParser   = require('body-parser');
+const http         = require('http');
+const urlParser    = require('url');
+const querystring  = require('querystring');
+const Router       = require('router');
+const bcrypt       = require('bcrypt');
+
+const router = new Router({ mergerParams: true });
+
+let messages = [];
+let nextId = 1;
+
+class Message {
```

Comment:
👍 I'll need to add a brief readme on classes in the first unit, but they should be intuitive for the students coming from Ruby.